zerver/lib/event_queue.py

```diff
@@ -343,7 +376,9 @@ def allocate_client_descriptor(new_queue_data):
     return client
 
 def do_gc_event_queues(to_remove, affected_users, affected_realms):
+    # type: (Set[str], Set[int], Set[int]) -> None
     def filter_client_dict(client_dict, key):
+        # type: (Dict, Union[int, str]) -> None
```

Comment:
I think `client_dict` should be typed as `MutableMapping[Union[int, str], ClientDescriptor]`.